bftengine/src/preprocessor/messages/PreProcessResultMsg.cpp

```diff
@@ -71,9 +71,8 @@ std::optional<std::string> PreProcessResultMsg::validatePreProcessResultSignatur
   auto hash = concord::util::SHA3_256().digest(requestBuf(), requestLength());
   std::unordered_set<bftEngine::impl::NodeIdType> seen_signatures;
   for (const auto& s : sigs) {
-    // insert returns std::pair<iterator, bool>. The bool indicates if the element was created, or it was already in the
-    // set and no insertion was performed. The latter case indicates that we already have got a signature from this
-    // replica.
+    // logic that filters duplicate signatures exist in the preprocessing phase,
```

Comment:
logic that filters duplicate signatures was performed in the ....
therefore duplication implies that the primary is probably malicious